### ISSUE TITLE
[: false: unexpected operator

### ISSUE THREAD

**Issue by hahaxd35** [ONLYOFFICE/DocumentServer#1088]
I have installed the new Version and i found this error:

/var/lib/dpkg/info/onlyoffice-documentserver-de.postinst: 127: [: false: unexpected operator

if [ ! $CLUSTER_MODE == true ]; then

**Comment by hahaxd35** [comment]
# restart dependent services
service supervisor restart >/dev/null 2>&1
service nginx restart >/dev/null 2>&1

still every error to /dev/null ......

**Comment by ShockwaveNN** [comment]
> I have installed the new Version and i found this error:

Which version are you installing? I think we fixed same error in 6.0.2

Are you facing same issue in 6.1.0?

**Comment by hahaxd35** [comment]
soory i have installed the 6.1.0 

ans this is only a post install error in Bash script

**Comment by hahaxd35** [comment]
oh i have forgot that :D

**Comment by ShockwaveNN** [comment]
Sooo, I'm not sure I'm understanding you're right

You got this error while installing 6.1.0?
Are you using clean install or upgrading from 6.0?
If upgrading from 6.0 - which version of 6.0?

**Comment by hahaxd35** [comment]
i have upgrades the Version 6.0.2-5
Unpacking onlyoffice-documentserver-de (6.1.0-83) over (6.0.2-5) ...

i have used this command:
apt update && documentserver-prepare4shutdown.sh && supervisorctl stop all && apt dist-upgrade

i think you maybe mean?
`if [ ! $CLUSTER_MODE  ]; then`
or 
`if [ $CLUSTER_MODE  ]; then`

**Comment by ShockwaveNN** [comment]
Yeah, seems something wrong, need some time to investigate

**Comment by ShockwaveNN** [comment]
Can confirm:
![image](https://user-images.githubusercontent.com/668524/101040712-42a98980-358d-11eb-9926-f9a3e9cef9fd.png)

Can be reproduced using this Dockerfile

```
FROM onlyoffice/documentserver-de:6.0.2.5

RUN rm -rf /etc/apt/sources.list.d/ds.list
RUN sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CB2DE8E5
RUN sudo echo "deb https://download.onlyoffice.com/repo/debian squeeze main" | sudo tee /etc/apt/sources.list.d/onlyoffice.list
RUN apt-get -y update
RUN service postgresql start && \
    service rabbitmq-server start && \
    service redis-server start && \
    apt-get -y install onlyoffice-documentserver-de
```

**Comment by ShockwaveNN** [comment]
Not 100% sure but seems this is same error as #1043 and we may forgotten to apply it's fix to 6.1.0 branch

**Comment by ShockwaveNN** [comment]
Correct line looks like:

```
if [ ! $CLUSTER_MODE = true ]; then
```

You can temporaly edit `/var/lib/dpkg/info/onlyoffice-documentserver-de.postinst` file and run installation again, until we figure out what is the best way to finally resolve this problem. Sorry for trouble

**Comment by ShockwaveNN** [comment]
I've created issue 47838 in our private bug tracker to discuss this situation and make sure this is last time we see this problem

Sorry again and thank you for your report

**Comment by hahaxd35** [comment]
ok i think:
if [ true ] would be better to read then If [ not false = true ]

but never mind

**Comment by ShockwaveNN** [comment]
@hahaxd35 I'll show your comment to dev team, but not 100% sure they apply it, I just copied text from patch which is included in 6.0.2

**Comment by ShockwaveNN** [comment]
DocuemntServer v 6.1.1 is released and this problem should be fixed
Sorry for same bug several times :(
